test-cases/tests/installation/a46-verify-migration-to-obo-was-successful.md

```diff
@@ -1,3 +1,10 @@
+---
+products:
+  - name: rhoam
+    targets:
+      - 1.38.0
```

Comment:
To me this was just one time test case for 1.38.0. I would tag it as "manual-selection" so we can only pick it again if needed (not expected though)